verifier/src/tcb.rs

```diff
@@ -216,38 +237,26 @@ impl<'a> TcbInfoRaw<'a> {
     ///     ```
     ///   or equivalent
     pub fn verify(self, key: &VerifyingKey, time: DateTime) -> Result<(), Error> {
-        self.verify_time(time)?;
         self.verify_signature(key)?;
+        let tcb_info = TcbInfo::try_from(&self)?;
+        tcb_info.verify_version()?.verify_time(time)?;
```

Comment:
Similar comment regarding creating a `.verify()` to encapsulate these two calls

Reply:
done in the most recent commit